app/src/main/java/io/github/muntashirakon/setedit/Utils/EditorUtils.java

```diff
@@ -1,7 +1,9 @@
-package io.github.muntashirakon.setedit;
+package io.github.muntashirakon.setedit.Utils;
```

Comment:
Package name usually contains lowercase letter to avoid confusing them with classes. Please refactor the files to use lowercase letters.

Reply:
Corrected, and I'm sorry about this beginner's error